ttnn/cpp/pybind11/operations/__init__.hpp

```diff
@@ -51,10 +52,14 @@ void py_module(py::module& module) {
 
     auto m_ternary_backward = module.def_submodule("ternary_backward", "ternary_backward operations");
     ternary_backward::py_module(m_ternary_backward);
-    
+
     auto m_unary_backward = module.def_submodule("unary_backward", "unary_backward operations");
     unary_backward::py_module(m_unary_backward);
 
+    auto m_ccl = module.def_submodule("ccl", "collective communication operations");
+    ccl::py_module_all_gather(m_ccl);
```

Comment:
why are these separate python modules instead of single `ccl`?

Reply:
Both are declared in two different files and if I use same name it is showing conflict so I just defined two modules for it

Reply:
these are not modules anymore, would be good to rename to smth like `ccl::py_bind_all_gather`